finaps/apps/mfing-website/src/app/app.component.ts

```diff
@@ -1,10 +1,12 @@
 import { Component } from '@angular/core';
-
 @Component({
   selector: 'finaps-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
```

Comment:
I reckon the css file can be removed and reminds me to set up scss before too much css is gonna be involved :D

Reply:
Agreed, removed from this specific file but implementing SCSS needs a new PR